modules/host/negotiaterecentrevision.go

```diff
@@ -62,17 +62,6 @@ func (h *Host) verifyChallengeResponse(fcid types.FileContractID, challenge cryp
 	if err != nil {
 		return nil, types.FileContractRevision{}, nil, err
 	}
-
-	// Sanity check - verify that the host has a valid revision and set of
-	// signatures.
-	h.mu.RLock()
-	blockHeight := h.blockHeight
-	h.mu.RUnlock()
-	err = modules.VerifyFileContractRevisionTransactionSignatures(recentRevision, revisionSigs, blockHeight)
```

Comment:
what makes this misplaced? The renter calls this function on the received values.